code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -121,6 +121,163 @@ All effects don't start immediately, but rather get worse over time; the rate is
+	color = "#d6b443"
+	nutriment_factor = 1 * REAGENTS_METABOLISM
+	boozepwr = 25
+	taste_description = "bath water"
+	glass_name = "glass of lager"
+	glass_desc = "A freezing pint of lager."
+
+/datum/reagent/consumable/ethanol/lagger_lager //ADDITION 01/19/2020 (Special Lager - WIP)
+	name = "Lagger_Lager"
+	description = "ERROR - DESCRIPTION NOT FOUND"
+	color = "#d6b443"
+	nutriment_factor = 1 * REAGENTS_METABOLISM
+	boozepwr = 25
+	taste_description = "glitchy"
+	glass_name = "glass of Lagger Lager"
+	glass_desc = "A freezing pint of Lagger Lager."
+
+/datum/reagent/consumable/ethanol/kvass //ADDITION 01/19/2020
+	name = "Kvass"
+	description = "A traditional fermented Slavic and Baltic beverage commonly made from rye bread."
+	color = "#b78315"
+	boozepwr = 2.5 //It's just soda to Russians
+	taste_description = "a warm summer day at babushka's cabin."
+	glass_name = "glass of kvass"
+	glass_desc = "A hearty glass of Slavic brew."
+
+/datum/reagent/consumable/ethanol/ipa //ADDITION 01/21/2020
+	name = "IPA"
+	description = "A type of light-colored beer similar to bitter, typically with a higher than average alcohol and hop content."
+	color = "#dfb653" // rgb: 42, 62, 87
+	boozepwr = 70 //slightly stronger than regular Ale
+	taste_description = "strong bitter flavor"
+	glass_name = "glass of IPA"
+	glass_desc = "A bitter pint of Indian Pale Ale."
+
+/datum/reagent/consumable/ethanol/guinness //ADDITION 01/21/2020
+	name = "Guinness"
+	description = "An alcoholic beverage from Old Earth. Helps you forget the potato famine."
+	color = "#dfb653" // rgb: 42, 62, 87
+	boozepwr = 70 //slightly stronger than regular Ale
+	taste_description = "Irish tap water"
+	glass_icon_state = "guinness_glass"
+	glass_name = "glass of Guinness"
+	glass_desc = "A tall pint of Guinness."
+
+/datum/reagent/consumable/ethanol/spacecastle //ADDITION 01/21/2020
+	name = "Spacecastle Black"
+	description = "An interesting brew that seems to have the apperance of a 'galaxy in a glass'."
+	color = "#181819" // rgb: 232, 6, 9
+	boozepwr = 65 //same as ale, but it makes you a bit sleepy
+	taste_description = "sky full of stars"
+	glass_icon_state = "spacecastle_glass"
+	glass_name = "glass of Spacecastle Black"
+	glass_desc = "A sparkling glass of Spacecastle Black."
+
+/datum/reagent/consumable/ethanol/hairy_dairy //ADDITION 01/21/2020
+	name = "The Hairy Dairy"
+	description = "A Manly Dorf... with cream?... WHO THOUGHT OF THIS!?!."
+	color = "#b6b0ae" // rgb: 182, 167, 174
+	boozepwr = 75 //For the (slightly less) manly only
+	quality = DRINK_GOOD
+	taste_description = "spoiled milk mixed with hair"
+	glass_icon_state = "hairy_dairy"
+	glass_name = "The Hairy Dairy"
+	glass_desc = "An awful concoction made from a Manly Dorf and cream..."
+
+/datum/reagent/consumable/ethanol/hairy_dairy/on_mob_life(mob/living/carbon/M)
+	M.adjustFireLoss(-8)
+	. = 1
+	..()
+
+/datum/reagent/consumable/ethanol/hairy_dairy/on_mob_metabolize(mob/living/carbon/M)
+	M.vomit (2)
```

Comment:
BRUH

hm yes today I will deliver an instant, 8 second hard stun with my hypospray, shotgun dart(s), syringe gun syringes, and/or damp rag (that also doubles as an incredible burn healing chem for plasmamen)

Reply:
Someone else came up with this drink idea and it needed a balance for the burn healing, this was their balance...